packages/paypal-commerce-integration/src/paypal-commerce-types.ts

```diff
@@ -34,11 +34,38 @@ export interface PayPalSDK {
         isEligible(): boolean;
         render(data: PayPalCommerceHostedFieldsRenderOptions): Promise<PayPalCommerceHostedFields>;
     };
+    Legal: PayPalLegal;
     Buttons(options: PayPalCommerceButtonsOptions): PayPalCommerceButtons;
     PaymentFields(options: PayPalCommercePaymentFieldsOptions): PayPalCommercePaymentFields;
     Messages(options: PayPalCommerceMessagesOptions): PayPalCommerceMessages;
 }
 
+interface PayPalLegalConfig {
+    fundingSource: string;
+}
+
+interface PayPalLegal {
+    (config: PayPalLegalConfig): PayPalLegal;
+    FUNDING: {
+        PAY_UPON_INVOICE: string;
+    };
+    render(container: string): void;
+}
+
+export interface RatePay {
+    birth_date: string | undefined;
```

Comment:
Why you use this syntax instead of `?` ?

Reply:
fixed